e2e/claims.service.e2e.ts

```diff
@@ -825,109 +825,103 @@ describe('Сlaim tests', () => {
-      await domainsService.createRole({
-        roleName: verifyVcRole,
-        namespace,
-        data: roles[`${verifyVcRole}.${root}`],
-        returnSteps: false,
-      });
-      const vc = await createExampleSignedCredential(
-        [],
-        `${verifyVcRole}.${root}`
-      );
-      const result = await claimsService.verifyVc(vc);
-      expect(result.errors).toHaveLength(0);
-      expect(result.isVerified).toBeTruthy;
-    });
-    test('resolveCredentialAndVerify should resolve a Verifiable Credential and call correct verification method (verifyVC)', async () => {
-      const roleName = `${resolveVC}.${root}`;
-      await signerService.connect(rootOwner, ProviderType.PrivateKey);
-      await domainsService.createRole({
-        roleName: resolveVC,
-        namespace,
-        data: roles[roleName],
-        returnSteps: false,
-      });
-      await signerService.connect(staticIssuer, ProviderType.PrivateKey);
-      await createExampleSignedCredential([], roleName);
-      const { issuedToken } = await enrolAndIssue(rootOwner, staticIssuer, {
-        subjectDID: rootOwnerDID,
-        claimType: roleName,
-        registrationTypes: [
-          RegistrationTypes.OnChain,
-          RegistrationTypes.OffChain, // role type issuer should have offchain claim
-        ],
-        publishOnChain: true,
-        issuerFields: [],
-      });
-      await signerService.connect(rootOwner, ProviderType.PrivateKey);
-      let subjectDoc = await didRegistry.getDidDocument({
-        did: rootOwnerDID,
-        includeClaims: true,
-      });
-      mockCachedDocument.mockResolvedValueOnce(subjectDoc);
-
-      await claimsService.publishPublicClaim({
-        claim: { token: issuedToken },
+    describe('Verify off-chain claim and credential', () => {
+      test('verifyVc should verify a VC with no errors if the issuer is authorized', async () => {
+        await signerService.connect(rootOwner, ProviderType.PrivateKey);
+        const vc = await createExampleSignedCredential(
+          [],
+          `${verifyVcRole}.${root}`
+        );
+        const result = await claimsService.verifyVc(vc);
+        expect(result.errors).toHaveLength(0);
+        expect(result.isVerified).toBeTruthy;
       });
+      test('resolveCredentialAndVerify should resolve a Verifiable Credential and call correct verification method (verifyVC)', async () => {
+        const roleName = `${resolveVC}.${root}`;
+        await signerService.connect(rootOwner, ProviderType.PrivateKey);
+        await signerService.connect(staticIssuer, ProviderType.PrivateKey);
+        await createExampleSignedCredential([], roleName);
+        const { issuedToken } = await enrolAndIssue(rootOwner, staticIssuer, {
+          subjectDID: rootOwnerDID,
+          claimType: roleName,
+          registrationTypes: [
+            RegistrationTypes.OnChain,
+            RegistrationTypes.OffChain, // role type issuer should have offchain claim
```

Comment:
This comment isn't needed here as root owner is not issue
```suggestion
            RegistrationTypes.OffChain,
```